src/doc/reference.md

```diff
@@ -1,4 +1,3 @@
-% The Rust Reference has moved
+# The Rust Reference has moved
 
-We've split up the reference into chapters. Please find it at its new
-home [here](reference/index.html).
+It is now hosted on [doc.rust-lang.org](https://doc.rust-lang.org/reference.html).
```

Comment:
This link points to the web, and the stable version, rather than locally, to whatever version we're building. On nightly, where this exists, this points to the right thing: https://doc.rust-lang.org/nightly/reference.html